non valid XML Content documentation (#72)
logstash-plugins/logstash-filter-xml: docs/index.asciidoc

Currently supported options are:

 - `strict` - forces the parser to fail early instead of accumulating errors when content is not valid xml.
 

> This still means that non allowed control characters like for instance ASCII 0x0 always mean a non valid XML.

Reviewer: ```suggestion
Control characters such as ASCII 0x0 are not allowed and _always_ result in non-valid XML.
```
